cron/poll_pull_requests.py

```diff
@@ -30,6 +30,12 @@ def poll_pull_requests(api):
         # file already exists, which is what we want
         pass
 
+    try:
+        mfp = open('server/meritocracy.json', 'x')
```

Comment:
I don't see the point in this. The later call to `open` with `w` will already create the file if it doesn't exist.